Dockerfile

```diff
@@ -86,3 +76,20 @@ RUN . /envfile && echo $ARCH && \
     chmod +x /usr/bin/kubeseal
 
 WORKDIR /apps
+
+# Run as unprivileged user
+RUN addgroup --system ${NONROOT_USER} && \
+    adduser -S -u ${NONROOT_USER_ID} ${NONROOT_USER} ${NONROOT_USER}
+USER ${NONROOT_USER_ID}
+
+# Helm plugins must be installed by the user
+RUN helm plugin install https://github.com/databus23/helm-diff && \
+    helm plugin install https://github.com/quintush/helm-unittest && \
+    helm plugin install https://github.com/chartmuseum/helm-push && \
+    rm -rf /tmp/helm-* \
+        /home/${NONROOT_USER}/.local/share/helm/plugins/helm-push/testdata \
+        /home/${NONROOT_USER}/.cache/helm/plugins/https-github.com-chartmuseum-helm-push/testdata
+
+# Run tests
+COPY test.sh /tmp/test.sh
+RUN /tmp/test.sh
```

Comment:
should not run `test.sh` in image.

test can be run on local image we built first